_components/input-mask/guidance/implementation.md

```diff
@@ -1,24 +1,84 @@
-The examples in the Preview section, above, are common input masks, but you can adapt the code to meet the needs of your project. In order to extend the functionality of the input mask, you must include a `placeholder` attribute as the visible input mask, using the underscore `_` to represent a character the user should enter. You may use punctuation literals like `[space]`, `-`, `/`, or `(` to help format your mask, but these literals will not be included in the final submitted value. You also need a related `pattern` attribute on the `input` element, its value epxressed as a regular expression that describes the intended format of the input data. Additionally, if the expected input is numeric, make sure to add the `inputmode="numeric"` attribute and setting so the numeric keypad is displayed. For example, an `input` with an input mask of three groups of three numerals, separated with spaces might have the following properties: 
+{:.usa-content-list }
+The examples in the Preview section above are common input masks, but you can adapt the code to meet the needs of your project. In order to extend the functionality of the input mask, you must include a `placeholder` attribute as the visible input mask, using the underscore `_` to represent a character the user should enter. You may use punctuation literals like `[space]`, `-`, `/`, or `(` to help format your mask, but these literals will not be included in the final submitted value. _See a complete list of format characters below_.
```

Comment:
Instead of giving directions for where to find the content, it would be better to provide a direct link to the section. To match format, it would also be good to have a content summary in bold. One suggestion:
```suggestion
**Customize the pattern.** The examples in the [component preview section]({{ site.baseurl }}/components/input-mask/#input-mask-preview-content) show common input mask patterns, but you can adapt the code to meet the needs of your project. In order to extend the functionality of the input mask, you must include a `placeholder` attribute as the visible input mask, using the underscore `_` to represent a character the user should enter. You may use punctuation literals like `[space]`, `-`, `/`, or `(` to help format your mask, but these literals will not be included in the final submitted value. A full list of formatting characters can be found in the [Available format characters table]({{ site.baseurl }}/components/input-mask/#available-format-characters).
```